### DISCUSSION TITLE
Hyperlinked table of contents

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#6826] [Help] [answered]
Hello,

Would it be possible to to create a hyperlinked table of contents or hyperlink to compnents on a form?

**Comment by melohagan** [comment, marked as answer]
Do you mean something like a **Scroll To Component** action?

Or maybe just hashed link anchors, e.g. https://docs.budibase.com/docs/rest-bindings#headers-example
But in the case of Budibase #headers-example would be the name of a component on that screen? 

Either way neither of these are currently available. There may be a way to do this with an Embed component.

**Reply by FlaminWrap** [reply]
Ah ok, nothing too important. I was going to include the documentation for an app within the app. I will find another way around it using form steps or something.
